x/swaprouter/client/cli/parse.go

```diff
@@ -0,0 +1,97 @@
+package cli
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+)
+
+// TODO: move these to exported types within an internal package
+type XCreatePoolInputs createBalancerPoolInputs
+
+type XCreatePoolInputsExceptions struct {
+	XCreatePoolInputs
+	Other *string // Other won't raise an error
+}
+
+type XCreateStableswapPoolInputs createStableswapPoolInputs
+
+type XCreateStableswapPoolInputsExceptions struct {
+	XCreateStableswapPoolInputs
+	Other *string // Other won't raise an error
+}
```

Comment:
Can you explain what is going on here / why the prefix?

Reply:
I don't know. This is a copy from gamm. Seems to be coming from this PR: https://github.com/osmosis-labs/osmosis/pull/3294

Based on the description, it inherits a code smell from the balancer. I wouldn't worry about these CLI issues since they are going to be replaced with Auto CLI soon